cmd/jaeger/internal/extension/jaegerstaticsampling/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
I think you're missing the point of the extension, but let me try to lead you to it this way - please write a README explaining what the business purpose of this extension is. As a user, why would I want to include it?

Reply:
done

Reply:
Everything you described in the readme (will be useful in the future) is what the extension does **internally**. If I am deploying Jaeger as an end user, why do I care that it does internally? How does it affect me as a user? The sampling strategies managed by the extension are not applied to the data passing through the collector, it's not what they are for.

Reply:
https://www.jaegertracing.io/docs/1.56/sampling/#remote-sampling